src/registrar/forms/application_wizard.py

```diff
@@ -351,13 +328,18 @@ class AboutYourOrganizationForm(RegistrarForm):
 
 
 class AuthorizingOfficialForm(RegistrarForm):
+    JOIN = "authorizing_official"
+
     def to_database(self, obj):
         if not self.is_valid():
             return
         contact = getattr(obj, "authorizing_official", None)
-        if contact is not None:
+        if contact is not None and not contact.has_more_than_one_join("authorizing_official"):
```

Comment:
```suggestion
        if contact is not None and not contact.has_more_than_one_join(self.JOIN):
```
Why not use this here, too?